fabtests/include/shared.h

```diff
@@ -69,6 +69,12 @@ extern "C" {
 	((type *) ((char *)ptr - offsetof(type, field)))
 #endif
 
+/*
+ * Internal version of deprecated APIs.
+ * These are used internally to avoid compiler warnings.
+ */
+#define OFI_MR_BASIC		(1 << 0)
+#define OFI_MR_SCALABLE		(1 << 1)
```

Comment:
As an alternative, consider:
```
#define OFI_MR_DEPRECATED_BITS (0x3)
```

Reply:
I was matching the strategy in libfabric to be clearer when reading but yeah we could do that for fabtests. I'm fine either way! You tell me.

Reply:
I think OFI_MR_DEPRECATED would be more annoying for you, so I'm leaning towards it.  My actual reason is for someone reading the code 2 years from now, ```~DEPRECATED``` may convey more meaning than ```~(BASIC | SCALABLE)```

Reply:
I prefer Sean's suggestion. The definitions are only used in the code to mask off unwanted bits so there is no need to keep them separate. 

This is different from the libfabric code, where the internal definition is still in place so that application backward compatibility can still be maintained.

Reply:
As you wish. Updated. I also removed FI_MR_LOCAL from the exclusion in the MR test because it didn't look necessary. I think maybe whoever wrote it was trying to exclude FI_LOCAL_MR? Unclear. You'll probably yell at me about it anyway so it'll be short lived :)